### ISSUE TITLE
ui-router-ng2 + angular-cli + lazyload

### ISSUE THREAD

**Issue by fpragana** [angular-ui/ui-router#3079]
I try use lazyload using _angular (2.0.2)_ + _angular-cli (1.0.0-beta.17)_ with _ui-router-ng2 (1.0.0-beta.3)_, but when call the state, the module is not found, i see this message in console: 

> Error: Cannot find module 'app/login/login.module'.

My folder structure is:
 myProject/src/app/login/login.module

I try use:
{ name: 'login', url: '/login',  lazyLoad: loadNgModule('/src/app/login/login.module')}
{ name: 'login', url: '/login',  lazyLoad: loadNgModule('src/app/login/login.module')}
{ name: 'login', url: '/login',  lazyLoad: loadNgModule('/app/login/login.module')}
{ name: 'login', url: '/login',  lazyLoad: loadNgModule('./app/login/login.module')}
{ name: 'login', url: '/login',  lazyLoad: loadNgModule('app/login/login.module')}

but always return Cannot find module

**Comment by christopherthielen** [comment]
Can you see what URL the browser fetches? In Chrome Dev Tools, open the "Network" tab and look for the 404 fetch.

**Comment by christopherthielen** [comment]
Ahh, I see that angular-cli uses webpack2's automatic code splitting feature.   For this to work, we need to a`System.import()` call in the routes file.    We also need support for more than a string as a parameter to the `loadNgModule` function.

There's a merged PR #3037 which adds this capability and will be released as `1.0.0-beta.4` fairly soon.

**Comment by christopherthielen** [comment]
When ui-router-ng2 1.0.0-beta.4 arrives: The magic incantation for lazy loading from angular-cli (with webpack 2) is:

```
{ 
  name: 'login', 
  url: '/login', 
  lazyLoad: loadNgModule(() => System.import('./app/login/login.module'))
}
```

closing via #3037
